### PR TITLE
[Fix] Fatal error: Uncaught Error: array_keys(): Argument #1 ($array) must be of type array

### PR DESCRIPTION
**Pull request by mebishalnapit**
This PR includes:

* Fixes PHP fatal error for exclude roles case when selected any on of them available previously and later unselected/deselected all of them

### CODE REVIEW

File: includes/common-pages.php [hunk 1]
```diff
@@ -205,8 +205,9 @@ public static function edit_page() {
 	 * @since 4.1.8 Added Exclude roles arguments.
 	 */
 	public static function activities_page() {
-		$roles = array_keys( ap_opt( 'activity_exclude_roles' ) );
-		$args  = array();
+		$roles_exclude = ap_opt( 'activity_exclude_roles' );
+		$roles         = ! empty( $roles_exclude ) ? array_keys( $roles_exclude ) : array();
```
Comment (rahularyan) [inline includes/common-pages.php:209]: @mebishalnapit This should be `is_array()` check instead of `!empty()`

**Review by rahularyan — CHANGES_REQUESTED** [review]

File: includes/common-pages.php (hunk 1 above)
Comment (mebishalnapit) [reply includes/common-pages.php]: @rahularyan Updated.

File: includes/common-pages.php [hunk 2]
```diff
@@ -205,8 +205,9 @@ public static function edit_page() {
 	 * @since 4.1.8 Added Exclude roles arguments.
 	 */
 	public static function activities_page() {
-		$roles = array_keys( ap_opt( 'activity_exclude_roles' ) );
-		$args  = array();
+		$roles_exclude = ap_opt( 'activity_exclude_roles' );
+		$roles         = ! is_array( $roles_exclude ) ? array_keys( $roles_exclude ) : array();
```
Comment (rahularyan) [inline includes/common-pages.php:209]: @mebishalnapit There is a bug in this line, condition should be opposite.

**Review by rahularyan — CHANGES_REQUESTED** [review]

File: includes/common-pages.php (hunk 2 above)
Comment (mebishalnapit) [reply includes/common-pages.php]: @rahularyan Fixed.

**Review by rahularyan — APPROVED** [review]

**Pull request merged**